### ISSUE TITLE
No se guarda la info al recargar

### ISSUE THREAD

**Issue by Alvax10** [juliantrovato18/desafio-7#8]
Juli, ahí me borra la info perfecto, pero fijate que cada vez que entro a la página me pide loguearme, estaría bueno que guarde los datos en el local y así no tengo que loguearme cada vez que entro, admeás al recargar en la página de mis mascotas se rompe (creo que en la home también pasa).

También no me muestra las mascotas cerca, a pesar de que reporté una que está en la misma ciudad que yo, no se que tan chico habrás puesto el radio de lat y lng pero te diría de agrandarlo un poco:

![Captura de pantalla de 2022-04-06 17-04-12](https://user-images.githubusercontent.com/64299010/162061730-1a92ee78-04e0-4f93-831d-9f3b2705c256.png)

**Comment by Alvax10** [comment]
También fijate que cuando hago el logueo, en la consola aparece un "Esta es la data: undefined" no se si eso será importante o no, vos fijate.

![Captura de pantalla de 2022-04-06 17-08-33](https://user-images.githubusercontent.com/64299010/162062360-214b15b1-f3c0-4a7f-bb2b-b0abf83e7da8.png)

Al final en la home al recargar creo que no falla, pero sí fijate lo de "mis mascotas"

**Comment by juliantrovato18** [comment]
no alva lo de la data no era importante era un endpoint que solo verifica el mail. la de mis mascotas reportardas no recarga?
el radio lo tengo en 10000 como decia marce

**Comment by juliantrovato18** [comment]
![imagen](https://user-images.githubusercontent.com/79176351/162075276-29f7b18f-9f0c-42c8-bf4f-bdf984247a70.png)

las mascotas me las muestra las cerca, decis de que pongas mas radio por las dudas?

**Comment by juliantrovato18** [comment]
y acabo de probar lo de las pages y al recargar andan.. probaste loggeandote de 0? te fijaste que guarda el localStorage? a mi me anda joya.. yo lo probe de 0 y en el local me guarda token, lat, lng pet Id user Id y recarga bien de hecho esa page me diste una mano vos que fue la primera que me explicaste lo de chequear la data

**Comment by Alvax10** [comment]
Buenas juli, sisi, agregale más radio por las dudas.

Si juli, me acuerdo que te ayudé con eso, por eso te digo, ahora voy a probar borrar el localStorage, hacerlo desde 0 y te aviso

**Comment by juliantrovato18** [comment]
dale lo subo a  200000 a mi me anda todo joya por eso es lo raro

**Comment by juliantrovato18** [comment]
ahi le agregue 20000 y encuentra muchas mas 🗡️

**Comment by Alvax10** [comment]
Borré todo el localStorage, me logueé, recargué la página y me llevó al logina devuelta. Hasta ahí todo bien, pero cuando recargo otra vez y me logue, me tira este error:

![Captura de pantalla de 2022-04-06 18-52-53](https://user-images.githubusercontent.com/64299010/162078117-19b1a605-a8ee-41ff-858f-7e96a017d49f.png)

**Comment by Alvax10** [comment]
> ahi le agregue 20000 y encuentra muchas mas dagger

Sii, ahí me apareció la mascota que reporté asique ya está bien el radio y esa página :D

**Comment by juliantrovato18** [comment]
ahi en lo de pets es, yo lo tengo asi si no tiene token ni lat ni lng /ingresar 
![imagen](https://user-images.githubusercontent.com/79176351/162078816-a112069e-035a-46bc-bd4e-df2a95afb7ed.png)

**Comment by juliantrovato18** [comment]
si no hay token, pero esta en localStorage que lo recupere  con me() y getMyPets()

**Comment by Alvax10** [comment]
> ahi en lo de pets es, yo lo tengo asi si no tiene token ni lat ni lng /ingresar ![imagen](https://user-images.githubusercontent.com/79176351/162078816-a112069e-035a-46bc-bd4e-df2a95afb7ed.png)

Claro, bueno seguramente es mi compu, asique voy a dejar que otro lo revise

**Comment by juliantrovato18** [comment]
y sino osea el ultimo else{
getMypets()
![imagen](https://user-images.githubusercontent.com/79176351/162079067-8286af9f-4d9d-42ba-87f4-aaa6765965da.png)

**Comment by juliantrovato18** [comment]
no, osea te pregunto si eso esta bien o falta otro chequeo?

**Comment by juliantrovato18** [comment]
porque por ahi se presenta alguna situacion mas que es la que vos recargas la page ahi y no agarra el token?
